app/index.js

```diff
@@ -309,6 +310,16 @@ app.on('ready', () => {
       }
     }));
 
+    // If we're on Mac make a Dock Menu
+    if (process.platform === 'darwin') {
+      const {app, Menu} = require('electron');
```

Comment:
`standard` needs spaces when de-structuring IIRC. Did the `git push` linting not complain?